### PR TITLE
misc. text fixes, ommitting trailing space, capitalization, etc.

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Not sure how much of this would apply to 2.2 too

### CODE REVIEW

File: mscore/instrdialog.cpp [hunk 1]
```diff
@@ -88,9 +88,8 @@ void InstrumentsDialog::on_saveButton_clicked()
             info.setFile(info.filePath() + ext);
       QFile f(info.filePath());
       if (!f.open(QIODevice::WriteOnly)) {
-            QString s = tr("Open Instruments File\n%1\nfailed: ")
-               + QString(strerror(errno));
-            QMessageBox::critical(mscore, tr("Open Instruments File"), s.arg(f.fileName()));
+            QString s = tr("Open Instruments File\n%1\nfailed: %2").arg(f.fileName()).arg(strerror(errno));
```
Comment (lasconic) [inline mscore/instrdialog.cpp]: If you do change this, I heard that it's more performant to use tr().arg(x,y,z) than tr().arg(x).arg(y).arg(z).

Comment (Jojo-Schmitz) [inline mscore/instrdialog.cpp]: Interesting, let me try that.

Comment (Jojo-Schmitz) [inline mscore/instrdialog.cpp]: Should I, while at it, do this for the entire code base?

Comment (lasconic) [inline mscore/instrdialog.cpp]: No. Touching the entire code base (just like this PR does in fact...) it's not good... it just mess up the entire history. To me the history is a lot more valuable than a white space too much or a QObject too much...

Comment (lasconic) [inline mscore/instrdialog.cpp]: So ideally, that should be done along the way, while fixing actual bugs and adding new features.

Comment (Jojo-Schmitz) [inline mscore/instrdialog.cpp]: in other words: not now, right?

Comment (lasconic) [inline mscore/instrdialog.cpp]: Does that make sense?

Comment (Jojo-Schmitz) [inline mscore/instrdialog.cpp]: Well, if it is a performance gain (and that is indeed very likely), it should be worth doing.

**Pull request merged**